hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -1047,5 +1048,9 @@ public void onFailure(Throwable t) {
     public void setOperationProvider(MapOperationProvider operationProvider) {
         this.operationProvider = operationProvider;
     }
+
+    public int getBackupCount() {
+        return mapConfig.getBackupCount();
```

Comment:
For this case we only need to implement `getBackupCount` which is used in `MapPartitionAwareService`(https://github.com/hazelcast/hazelcast/pull/8105/files#diff-5b16faeb24210a5b889ca49c7a666bc4), where the target object changed from `MapContainer` to `MapProxyImpl`.